tests/parameter_sources/elasticlogs_kibana_source_test.py

```diff
@@ -45,6 +45,23 @@ def test_create_discover(time):
     assert response == load("discover")
 
 
+@mock.patch("time.time")
+@mock.patch.dict("eventdata.utils.globals.global_fieldstats", {"elasticlogs-*_@timestamp": {"max": 1573430400000, "min": 1573344000000}})
```

Comment:
Ideally we'd use a fixture here (see e.g. https://github.com/elastic/rally/blob/6e6ce1cde08effb32d1c9dc9a505971799d479b4/it/proxy_test.py#L31-L43 for a more complex example) but it would also be ok if we would set the value at the beginning of the test and reset it at the end (in a `try ... finally` to ensure it is reset also when the test fails).

Reply:
added fixure